src/Form/Type/ImmutableTabsType.php

```diff
@@ -24,14 +24,18 @@
  *
  * @author Benoit de Jacobet <benoit.de-jacobet@ekino.com>
  */
-class ImmutableTabsType extends ImmutableArrayType
+class ImmutableTabsType
 {
-    /**
-     * {@inheritdoc}
-     */
+    private $immutableArrayType;
+
+    public function __construct(ImmutableArrayType $immutableArrayType)
+    {
+        $this->immutableArrayType = $immutableArrayType;
+    }
+
     public function configureOptions(OptionsResolver $resolver)
     {
-        parent::configureOptions($resolver);
+        $this->immutableArrayType->configureOptions($resolver);
 
         $resolver->setRequired('tabs');
```

Comment:
Improvement: the `tabs` value should be validated at least with an allowed type.

Reply:
okay I'll implement configureOptions if I go back to inheritance then